indico/modules/events/contributions/contrib_fields.py

```diff
@@ -37,15 +38,22 @@ class ContribFieldConfigForm(IndicoForm):
     title = StringField(_('Title'), [DataRequired()], description=_("The title of the field"))
     description = TextAreaField(_('Description'), description=_("The description of the field"))
     is_required = BooleanField(_('Required'), widget=SwitchWidget(),
-                               description=_("Whether the user has to fill out the field"))
+                               description=_("Whether the field has to be filled out"))
     is_active = BooleanField(_('Active'), widget=SwitchWidget(),
                              description=_("Whether the field is available."),
                              default=True)
+    visibility = IndicoEnumRadioField(_('Visibility'), [DataRequired()], default=ContributionFieldVisibility.public,
+                                      enum=ContributionFieldVisibility,
+                                      description=_('Who will be able to see the field'))
```

Comment:
at least to me it doesn't sound right, maybe something like: "Determines who can fill in the field"

Reply:
But it actually doesn't determine who will be able to fill the field, but who will be able to see the value of the field. How about 'Determines who can see the value of the field'?